Tests/Library/EnvelopeFactoryTests.ts

```diff
@@ -235,4 +235,32 @@ describe("Library/EnvelopeFactory", () => {
 
         });
     });
+
+    describe("MetricData", () => {
+        let metricTelemetry: Contracts.MetricTelemetry;
+        beforeEach(() => {
+            metricTelemetry = {
+                name: "TestName",
+                value: 123,
+                namespace: "TestNamespace",
```

Comment:
Is this something we should be adding to the web code as well?
We only have these
export class DataPoint {

    /**
     * Name of the metric.
     */
    public name: string;

    /**
     * Metric type. Single measurement or the aggregated value.
     */
    public kind: DataPointType = DataPointType.Measurement;

    /**
     * Single value for measurement. Sum of individual measurements for the aggregation.
     */
    public value: number;

    /**
     * Metric weight of the aggregated metric. Should not be set for a measurement.
     */
    public count: number;

    /**
     * Minimum value of the aggregated metric. Should not be set for a measurement.
     */
    public min: number;

    /**
     * Maximum value of the aggregated metric. Should not be set for a measurement.
     */
    public max: number;

    /**
     * Standard deviation of the aggregated metric. Should not be set for a measurement.
     */
    public stdDev: number;
}

Reply:
Yes, you should, "ns" is the actual field we put in the envelope, namespace is the property name of the class we expose in the API

Reply:
OK, will raise an issue on AI-JS